pilot/pkg/security/authz/builder/testdata/http/allow-full-rule-out.yaml

```diff
@@ -51,29 +51,41 @@ typedConfig:
             - orRules:
                 rules:
                 - header:
-                    exactMatch: method
                     name: :method
+                    stringMatch:
+                      ignoreCase: false
+                      exact: method
                 - header:
                     name: :method
-                    prefixMatch: method-prefix-
+                    stringMatch:
+                      ignoreCase: false
```

Comment:
Is this a behavior change, or was the deprecated prefixMatch on method case insensitive ?

Reply:
+1, seems suscipious.

Reply:
Oh actually.. ignoreCase=false means "case matters". Double negative confused me.

We should double check but this looks right to me.